docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -0,0 +1,207 @@
+```python
+assets = kili.assets("<your_project_id>", fields=["externalId", "labels.jsonResponse"], label_consensus_mark_gt=0.7)
+# + asset conversion code
+```
+
+**Get all the labels with a consensus mark above 0.7**
+```python
+labels = kili.labels("<your_project_id>", fields=["labelOf.externalId", "jsonResponse"], consensus_mark_gt=0.7)
+# + label conversion code
+```
+
+**Get all the labels done by a given author**
+```python
+labels = kili.labels("<your_project_id>", fields=["labelOf.externalId", "jsonResponse"], author_in=["John Smith"])
+# + label conversion code
+```
+will directly return a list of labels authored by John Smith. In the `author_in` , you can also pass the first name, the last name, or the first name + last name of the user for which you want to fetch the labels.
+
+
+### Filtering specific labels per asset through the label properties
+You can also look for specific labels, for example the last "review" status label per user, and dump the result into a json file. You can use the field `"labels.isLatestReviewLabelForUser"` to know if the label is the latest per user.
+```python
+assets = kili.assets("<your_project_id>", fields=["externalId", "labels.jsonResponse", "labels.isLatestReviewLabelForUser"])
+
+for asset in assets:
+    if asset["labels"]: # covers the assets without annotations
+        for label in asset["labels"]:
+            if label["isLatestReviewLabelForUser"] and "JOB_0" in label["jsonResponse"]:
+                annotation = label["jsonResponse"]["JOB_0"]
+                with (Path("/tmp") / (asset["externalId"] + ".json")).open("w", encoding="utf-8") as f:
+                    f.write(json.dumps(annotation))
+                break # once we find a latest label done by a reviewer, we move on to the next asset.
+```
+
+## Exporting a whole project
+There is also a method to export a full project into specific export formats. It can be useful when your goal is to use a standard export format.
+
+### Available formats
+
+Here are the available formats:
+
+| Format        | UI  | Python Client | Command Line Interface |
+| ------------- | --- | ------------- | ---------------------- |
+| Kili (raw)    | ✅   | ✅             | ✅                      |
+| Kili (simple) | ✅   | ❌             | ❌                      |
+| YOLO V4       | ✅   | ✅             | ✅                      |
+| YOLO V5       | ✅   | ✅             | ✅                      |
+| YOLO V7       | ❌   | ✅             | ✅                      |
+| Pascal VOC    | ✅   | ❌             | ❌                      |
+| COCO          | ❌   | ✅             | ✅                      |
+
+
+### The `.export_labels` method
+
+The `.export_labels` method enables the export of a full project. It does the following preprocessing:
+
+* Only fetches the labels of types `"DEFAULT"` and `"REVIEW"`.
+* If specified, only selects a subset of asset ids.
+* You can specify a standard export format (only available for a restricted set of ML task).
```

Comment:
You can specify a standard export format -> Exports labels to one of the standard formats

This will make it grammatically in line with the other points